src/test/java/com/dnsimple/DomainsTest.java

```diff
@@ -80,18 +80,19 @@ public void testGetDomain() throws DnsimpleException, IOException {
     server.stubFixtureAt("getDomain/success.http");
 
     Domain domain = client.domains.getDomain("1", "example.com").getData();
-    assertThat(domain.getId(), is(1));
-    assertThat(domain.getAccountId(), is(1010));
-    assertThat(domain.getRegistrantId(), is(0));
+    assertThat(domain.getId(), is(181984));
+    assertThat(domain.getAccountId(), is(1385));
+    assertThat(domain.getRegistrantId(), is(2715));
     assertThat(domain.getName(), is("example-alpha.com"));
     assertThat(domain.getUnicodeName(), is("example-alpha.com"));
-    assertThat(domain.getToken(), is("domain-token"));
-    assertThat(domain.getState(), is("hosted"));
+    assertThat(domain.getToken(), isEmptyOrNullString());
```

Comment:
`token` is actually gone since a while. We should likely remove this setter/getter.